modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to `ClusterRoleBindings`
+
+As a cluster administrator, you can add unauthenticated users to the following ClusterRoleBindings in {product-title}.
+
+. `system:scope-impersonation`
+. `system:webhook`
+. `system:oauth-token-deleter`
+. `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying role bindings, particularly when including unauthenticated access.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* The OpenShift CLI (oc) is installed.
+
+.Procedure
+
+.Add unauthenticated groups to a `ClusterRoleBinding`
+
+. Create a YAML file named `add-<cluster-role-binding>-unauth.yaml` and add the following content:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+ annotations:
+   rbac.authorization.kubernetes.io/autoupdate: "true"
+ name: restore-<cluster-role-binding>-unauth
```

Comment:
For both of these places that call out to replace this with the cluster role binding, I'd add a callout or something to say to make sure to replace them with the one you want to update.